sunpy/tests/tests/test_mocks.py

```diff
@@ -16,17 +16,17 @@ def test_MockObject_illegal_kwargs(mocked_mockobject):
     Any attempt to use a kwarg which has the same name as an attribute/method
     of the underlying object or datastore will raise a ValueError.
     """
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="kwarg 'values' is already an attribute of <class 'dict'> or <class 'sunpy.tests.mocks.MockObject'>"):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="kwarg 'values' is already an attribute"):
```
Same for the others below.

Reply:
Done. Made similar changes to test_MockObject_illegal_kwargs(mocked_mockobject) function. Thank You.